src/actions/summit-actions.js

```diff
@@ -93,7 +94,7 @@ export const getSummitById = (summitId) => async (dispatch, getState) => {
   });
 };
 
-export const setCurrentSummit = (summit) => async (dispatch, getState) => {
+export const setCurrentSummit = (summit) => async (dispatch) => {
```

Comment:
what is the diff of this ( setCurrentSummit) vs getSummitById
seems to be querying the same data